angular.json

```diff
@@ -158,5 +158,8 @@
       }
     }
   },
-  "defaultProject": "teammates"
-}
+  "defaultProject": "teammates",
+  "cli": {
+    "analytics": false
+  }
+}
```

Comment:
unnecessary changes? u can disable angular analytics globally with  `ng analytics off` so you don't get these changes